packages/preview/ssrn-scribe/0.4.8/README.md

```diff
@@ -0,0 +1,92 @@
+# Typst-Paper-Template
+
+Following the official tutorial, I create a single-column paper template for general use. You can use it for papers published on SSRN etc.
+
+## Updates
+
+* v0.4.5: Adjust the template and ready to the package manager of Typst.
+  * `bibloc`, `bibstyle` and `bibtitle` has been integrated into the bibliography function.
+
+## How to use
+
+1. Download the template or clone the repository.
+
+<!-- (**If you use the latest version `v0.11.0`, you can use `typst init @preview/general-paper-template` to generate the `main.typ` template directly.**) -->
+
+2. generate your bibliography file using `.biblatex` and store the file in the same directory of the template.
+3. modify the `main.typ` file and compile it.
+   ***Note:* You should have `paper_template.typ` and `main.typ` in the same directory.**
```

Comment:
These steps should be updated to reflect how to use the packaged version.

Reply:
Updated the readme file with the instructions for using the package manager.